oeps/best-practices/oep-0066-bp-authorization.rst

```diff
@@ -0,0 +1,433 @@
+OEP-66: User Authorization
+###########################
+
+.. list-table::
+   :widths: 25 75
+
+   * - OEP
+     - :doc:`OEP-66 <oep-0066-bp-authorization>`
+   * - Title
+     - User Authorization
+   * - Last Modified
+     - 2023-10-02
+   * - Authors
+     - Hilary Sinkoff (hsinkoff@2u.com), Jeremy Bowman (jbowman@edx.org)   
+   * - Arbiter
+     - Feanil Patel
+   * - Status
+     - Draft
+   * - Type
+     - Best Practice
+   * - Created
+     - 2023-08-21
+   * - `Review Period`
+     - 2023-09-27 - 2023-10-18
+   * - Replaces
+     - :doc:`oep-0009-bp-permissions`
+   * - References
+     - 
+       * :doc:`/architectural-decisions/oep-0004-arch-oauth-scopes`
+       * :doc:`oep-0042-bp-authentication`
+     
+Summary
+*******
+
+There are a variety of manners in which authorization is handled within the Open edX ecosystem. 
+The goal of OEP-66 is to provide best practices that should be used with all the 
+systems/protocols and outline the currently in use systems/protocols.
+
+Motivation
+***********
+
+Best Practices Motivation
+--------------------------
+To date, the implementation and verification of permissions have been somewhat
+conflated in the edX codebase.  When a user attempts an action which is not
+permitted for all users, the code typically directly checks properties of the
+user: are they a superuser, do they belong to a particular group or have a
+particular role, etc.  This has a few drawbacks:
+
+* This is often a violation of DRY ("Don't Repeat Yourself") which results in
+  the same basic permission check being copied in multiple code locations,
+  making it very difficult to consistently change its implementation (as may
+  happen when a new type of user is introduced, or the need for a special
+  exception becomes clear).
+* Fine-grained permission checks have sometimes been avoided even when
+  appropriate due to the difficulty of copying the permission code around,
+  finding a common place to store it, or updating all the code that used a
+  coarser initial implementation.  This has resulted in some users being
+  technically capable of performing actions which logically should not be
+  permitted.
+* When a decision is made to change who is granted a particular permission,
+  it can be difficult to avoid accidentally changing other permissions with
+  a similar implementation.
+
+In Use System/Protocol Motivation
```

Comment:
This title confuses me and I don't know what even to suggest. Is it something like, "Currently Used Systems/Protocols"?